### PR TITLE
Try to get the testing working again.

### PR DESCRIPTION
**Pull request by wxtim**
Replace the unit testing and linting tests from Travis with GH actions.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,13 +18,6 @@ after_failure:
 
 jobs:
   include:
-    - name: "Unit Tests"
-      install:
-        - now install coverage linters pytest cylc rose
-      script:
-        - now test style
-        - now test units
```
Comment (oliver-sanders) [inline .travis.yml:26]: Why not just copy the `now` commands over to GH actions to save re-implementing everything?

Comment (wxtim) [reply .travis.yml:26]: Will that work?

File: .travis/now [hunk 2]
```diff
@@ -75,6 +75,7 @@ _gh_extract () {  # extract project from GitHub to $HOME
 }
 
 _install_coverage () {
+    APT+=(nodejs)
```
Comment (oliver-sanders) [inline .travis/now:78]: Python-Coverage doesn't use nodejs.

File: .travis.yml (hunk 1 above)
Comment (oliver-sanders) [reply .travis.yml:26]: Yes that will work.

Comment (wxtim) [reply .travis.yml:26]: I'll give it a go - probably will end up creating a new PR and closing this one. Ah well.

Comment (oliver-sanders) [reply .travis.yml:26]: What's the motive for this PR?

Comment (oliver-sanders) [reply .travis.yml:26]: Happy to move over to GH actions, however, it wont help the tests to pass.

Comment (wxtim) [reply .travis.yml:26]: Tests take a long time to pass or fail. Perhaps your are right - i should get it working on Travis.

**Pull request closed without merging**